### DISCUSSION TITLE
Only HBO Max doesn’t play for me. What could me wrong?

### DISCUSSION THREAD

**Discussion by Pipole26** [Eloston/ungoogled-chromium#1709] [Q&A]
I opened this issue here to not mix up with more serious problems.

It says it can’t play and I might be in Incognito Mode. 

With a fresh install and no modifications (besides placing the widevine folder in the right place) the same issue occurs.

Netflix and Amazon runs smoothly.

Thank you.

Edit: To be more specific, it apparently detects I've Widevine, allows me to get to the point where I click to watch something, but when it's time to play it says "Can't Play Title" and "make sure your browser is not in private browsing". 

I looked for it on the web and some people said resetting permissions for the site fixed it, but I had no such luck. I tried disabling all extensions and fingerprinting options and still no luck.

**Comment by wchen342** [comment]
Does anything show up in the developer console?

**Reply by Pipole26** [reply]
> Does anything show up in the developer console?

Sorry for the last response.

https://i.imgur.com/JzioEXi.png

Also one update on Amazon: it doesn't run HD. Netflix apparently runs normally.

**Reply by Pipole26** [reply]
My problem was never fixed.

**Reply by networkException** [reply]
If widevine is loaded correctly (also no other modifications or adblockers) and it still doesn't work thats likely a duplicate of https://github.com/ungoogled-software/ungoogled-chromium/issues/1258
